src/install/installer.ts

```diff
@@ -64,8 +74,8 @@ async function validateCache(packagePath: string, browsersPath: string, linksDir
   let downloadedBrowsers = (await fsReaddirAsync(browsersPath)).map(file => path.join(browsersPath, file));
   downloadedBrowsers = downloadedBrowsers.filter(file => browserPaths.isBrowserDirectory(file));
   const directories = new Set<string>(downloadedBrowsers);
-  for (const browser of allBrowsers)
-    directories.delete(browserPaths.browserDirectory(browsersPath, browser));
+  for (const browserPath of usedBrowserPaths)
+    directories.delete(browserPath);
   for (const directory of directories) {
     logPolitely('Removing unused browser at ' + directory);
```

Comment:
Maybe update the wording here to mention that the downloaded browser may be corrupted?

Reply:
99% of the time it will be unused - so I'd keep it as-is.